### PR TITLE
[WIP] Install and configure pulp-container plugin

### LINKED ISSUE #254: Infra: Install pulp_container plugin
Install [pulp_container](https://github.com/pulp/pulp_container) plugin so that we can have a consistent way to build images both on and off OSD. In the future, pulp_container also allows us to offer an image registry service to PAH users.

### PR DESCRIPTION
**Pull request by bmclaughlin**
Installed `pulp-container` via PyPI and configured per the steps in [1].  Dev environment still works as expected and output from configuration steps appear in the logs when building the environment as well. 

[1] https://pulp-container.readthedocs.io/en/latest/installation.html#enable-oci-container-image-building

Fixes https://github.com/ansible/galaxy-dev/issues/254.

### CODE REVIEW

**Comment by bmclaughlin** [comment]
Extra commit is from the Postgres issue that I ran into earlier, see #259

File: docker/pulp/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM centos:8
```
Comment (cutwater) [inline docker/pulp/Dockerfile]: Changing major version of container is probably out of scope of this change?

File: docker-compose.yaml [hunk 2]
```diff
@@ -56,6 +56,8 @@ services:
     env_file:
       - ./docker/pulp/env
     command: ['run', 'worker']
+    devices:
```
Comment (cutwater) [inline docker-compose.yaml]: I don't think we have access to `/dev/fuse` in OpenShift environment. Can you please elaborate?

**Review by cutwater — CHANGES_REQUESTED** [review]

File: docker/pulp/Dockerfile (hunk 1 above)
Comment (bmclaughlin) [reply docker/pulp/Dockerfile]: Moving this to a separate PR.

File: docker-compose.yaml (hunk 2 above)
Comment (bmbouter) [reply docker-compose.yaml]: This is a buildah requirement for building so if it's not available in OpenShift then it won't run there. We need to figure out if OpenShift doesn't have this due to it being too new or if it's omitted by design. Who from that group can help us answer that?

**Comment by bmclaughlin** [comment]
Replaced by https://github.com/ansible/galaxy_ng/pull/147

**Pull request closed without merging**